diff --git a/node_modules/@theguild/components/dist/index.js b/node_modules/@theguild/components/dist/index.js
index 76a6979..6ef5653 100644
--- a/node_modules/@theguild/components/dist/index.js
+++ b/node_modules/@theguild/components/dist/index.js
@@ -2718,7 +2718,7 @@ var Navigation$1 = _styled__default["default"].nav(function (_ref) {
     "right": "0px",
     "bottom": "0px",
     "left": "0px"
-  }, react.css(_templateObject3$6 || (_templateObject3$6 = _taggedTemplateLiteral(["\n          z-index: 300; // TODO: Used for Docusaurus, remove when no longer needed.\n        "]))), !isModalOpen && react.css(_templateObject4$5 || (_templateObject4$5 = _taggedTemplateLiteral(["\n            top: -100vh;\n            bottom: 100vh;\n          "])))])];
+  }, react.css(_templateObject3$6 || (_templateObject3$6 = _taggedTemplateLiteral(["\n          z-index: 300; // TODO: Used for Docusaurus, remove when no longer needed.\n        "]))), !isModalOpen && react.css(_templateObject4$5 || (_templateObject4$5 = _taggedTemplateLiteral(["\n            top: -100%;\n            bottom: 100vh;\n          "])))])];
 });
 var Controls$1 = _styled__default["default"].menu(function () {
   return [{
