diff --git a/dist/index.js b/dist/index.js
index b89506b4f924e3063cc7bb67d7ad1d6dda85d51d..cd0ed4e28d6c8e495c9a546e40a8c8f9462964bd 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -3171,12 +3171,21 @@ var InnerLayout = ({
     flatDocsDirectories,
     directories
   } = useMemo5(
-    () => normalizePages({
-      list: pageMap,
-      locale,
-      defaultLocale,
-      route: fsPath
-    }),
+    () => {
+      const res = normalizePages({
+        list: pageMap,
+        locale,
+        defaultLocale,
+        route: fsPath
+      })
+      if (fsPath.startsWith('/blog/tag/')) {
+        res.activeThemeContext.layout = 'full'
+        res.activeThemeContext.sidebar = false
+        res.activeThemeContext.timestamp = false
+        res.activeThemeContext.pagination = false
+      }
+      return res
+    },
     [pageMap, locale, defaultLocale, fsPath]
   );
   const themeContext = __spreadValues(__spreadValues({}, activeThemeContext), frontMatter);
