# todo: fix in nextra
diff --git a/dist/index.js b/dist/index.js
index 5eb1b039016caec28fc416db57592f44b2bc24a6..97fc1d28b6c0401e737d978e99a7a9eab835fe0b 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -3181,12 +3181,21 @@ function useDirectoryInfo(pageMap) {
   const { locale = DEFAULT_LOCALE, defaultLocale, asPath } = useRouter9();
   return useMemo5(() => {
     const fsPath = getFSRoute(asPath, locale);
-    return normalizePages({
+    const res = normalizePages({
       list: pageMap,
       locale,
       defaultLocale,
       route: fsPath
     });
+
+    if (asPath.startsWith('/blog/tag/')) {
+      res.activeThemeContext.layout = 'full'
+      res.activeThemeContext.sidebar = false
+      res.activeThemeContext.timestamp = false
+      res.activeThemeContext.pagination = false
+    }
+
+    return res
   }, [pageMap, locale, defaultLocale, asPath]);
 }
 var classes5 = {
