diff --git a/dist/index.js b/dist/index.js
index 72788625f8c8e1140a30ba703413b18d0e861802..7bedebb21a717c0b612cadce3d959a5e36f3b1a8 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -3115,12 +3115,21 @@ var InnerLayout = ({
     flatDocsDirectories,
     directories
   } = useMemo5(
-    () => normalizePages({
-      list: pageMap,
-      locale,
-      defaultLocale,
-      route: fsPath
-    }),
+    () => {
+      const res = normalizePages({
+        list: pageMap,
+        locale,
+        defaultLocale,
+        route: fsPath
+      })
+      if (fsPath.startsWith('/blog/tag/')) {
+        res.activeThemeContext.layout = 'full'
+        res.activeThemeContext.sidebar = false
+        res.activeThemeContext.timestamp = false
+        res.activeThemeContext.pagination = false
+      }
+      return res
+    },
     [pageMap, locale, defaultLocale, fsPath]
   );
   const themeContext = __spreadValues(__spreadValues({}, activeThemeContext), frontMatter);
