diff --git a/dist/mermaid-isomorphic.js b/dist/mermaid-isomorphic.js
index d56f2269ac90a3bb9f3073fb4cbd6b4ffa804e69..59b6e849451215a164e6373ab77ebfd769cf0519 100644
--- a/dist/mermaid-isomorphic.js
+++ b/dist/mermaid-isomorphic.js
@@ -1,4 +1,5 @@
-import { chromium } from 'playwright';
+import { chromium as playwright } from 'playwright';
+import chromium from '@sparticuz/chromium';
 const html = import.meta.resolve('../index.html');
 const mermaidScript = {
     url: import.meta.resolve('mermaid/dist/mermaid.js')
@@ -7,6 +8,10 @@ const faStyle = {
     // We use url, not path. If we use path, the fonts can’t be resolved.
     url: import.meta.resolve('@fortawesome/fontawesome-free/css/all.css')
 };
+
+chromium.setGraphicsMode = false;
+const useSparticuzChromium = process.env.CI && process.platform === 'linux';
+const executablePathPromise = useSparticuzChromium ? chromium.executablePath() : undefined;
 /* c8 ignore start */
 /**
  * Render mermaid diagrams in the browser.
@@ -122,7 +127,21 @@ async function getBrowser(browserType, launchOptions) {
  *   A function that renders Mermaid diagrams in the browser.
  */
 export function createMermaidRenderer(options = {}) {
-    const { browserType = chromium, launchOptions } = options;
+    let { browserType = playwright, launchOptions } = options;
+
+    if (useSparticuzChromium) {
+        if (options.browserType) {
+            throw new Error('options.browserType is not supported because @hasparus patched it to run on Vercel builds, sorry');
+        }
+        browserType = {
+            launch: async () => playwright.launch({
+                ...launchOptions,
+                args: chromium.args,
+                executablePath: await executablePathPromise,
+            })
+        }
+    }
+
     let browserPromise;
     let count = 0;
     return async (diagrams, renderOptions) => {
